ci/travis/ci.sh

```diff
@@ -233,21 +233,27 @@ install_go() {
   fi
 }
 
-bazel_ensure_buildable_on_windows() {
+_bazel_build_before_install() {
+  local target
   if [ "${OSTYPE}" = msys ]; then
-    # This performs as full of a build as possible, to ensure the repository always remains buildable on Windows.
+    # On Windows, we perform as full of a build as possible, to ensure the repository always remains buildable on Windows.
     # (Pip install will not perform a full build.)
-    # NOTE: Do not add build flags here. Use .bazelrc and --config instead.
-    bazel build -k "//:*"
+    target="//:*"
+  else
+    # Just build Python on other platforms.
+    # This because pip install captures & suppresses the build output, which causes a timeout on CI.
+    target="//:ray_pkg"
   fi
+  # NOTE: Do not add build flags here. Use .bazelrc and --config instead.
+  bazel build -k "${target}"
 }
 
 install_ray() {
   # TODO(mehrdadn): This function should be unified with the one in python/build-wheel-windows.sh.
   (
     cd "${WORKSPACE_DIR}"/python
     build_dashboard_front_end
-    pip install -v -v -e .
+    pip install -v -e .
```

Comment:
how about wrap it inside keep_alive script just to be safe?

Reply:
Yeah maybe that's a good idea, thanks. I think I'll also move the build command above this one so that it's obvious they're paired together.